Example for changing viewer position

smrghsh opened the pull request:
Added simple example for changing viewer position. 
@stalgiag I'm having trouble getting this example to load in the docs for both firefox and chrome-- but I am unsure if this issue is just on my machine and not on others. In the current version of this draft, I've included the sample in `tests\manual-test-examples\p5vr\scratchpad` and will remove this later when the sketch loads in the docs.

stalgiag commented:
Your sketch code wasn't working. As for it working in the docs, your jsdoc file is messed up. Seems like a find and replace gone wrong.

I rewrote your sketch and added it to the jsdoc example for [setViewerPosition in this branch](https://github.com/stalgiag/p5.xr/tree/smrghsh-viewerposition-example).

Tested and working in the docs, in headset, in latest Firefox and Chrome.

smrghsh commented:
Thank you, will examine.

On Sun, Sep 11, 2022 at 11:43 AM Stalgia Grigg ***@***.***>
wrote:

> Your sketch code wasn't working. As for it working in the docs, your jsdoc
> file is messed up. Seems like a find and replace gone wrong.
>
> I rewrote your sketch and added it to the jsdoc example for setViewerPosition
> in this branch
> <https://github.com/stalgiag/p5.xr/tree/smrghsh-viewerposition-example>.
>
> Tested and working in the docs, in headset, in latest Firefox and Chrome.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/stalgiag/p5.xr/pull/188#issuecomment-1243000982>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFNSQU7EYVAL56MTKUVOMXTV5YD4XANCNFSM6AAAAAAQJRBW5M>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
